src/nvim/eval.c

```diff
@@ -9547,6 +9547,40 @@ static void f_getbufvar(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   }
 }
 
+// "getchangelist()" function
+static void f_getchangelist(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+{
+  tv_list_alloc_ret(rettv, 2);
+  (void)tv_get_number(&argvars[0]);  // issue errmsg if type error
```

Comment:
Is this actually required for Neovim to issue an error message? I left it WIP because of this and the compiler warning from gcc for an unused return value.

Reply:
Very weird.  I guess that's why `emsg_off++`  is needed after.

Can assign it to the `vim_ignored` global, to avoid the failed CI.

If the tests pass then let's leave it at that :)